.github/workflows/test.yml

```diff
@@ -1,9 +1,9 @@
 name: Test colcon-ros-bundle
 on:
   pull_request:
-  # Runs every morning 08:43AM PST
+  # Run tests hourly
   schedule:
-    - cron: '43 17 * * *'
+    - cron: '0 * * * *'
```

Comment:
Do you want to RNG a minute offset?

Reply:
I don't think we can do that with the GH workflow syntax. (Unless you're suggesting to just put a random minute).
Plus, we need these tests to run consistently every hour.

Reply:
@piraka9011 you'll still need the suggestion per [crontab guru](https://crontab.guru/#43_17_*_*_*).